Modules/JavaScript/Classes-Constructor-New/homework.js

```diff
@@ -0,0 +1,86 @@
+console.log(`
+  Task.
+
+    Create a Vector object that supports:
+     1. add (same length)
+     2. subtract
+     3. dot
+     4. norm
+     5. If you try to add, subtract, or dot two vectors with different lengths, you must throw an error.
+     6. toString
+     7. equals
+`);
+
+class Vector {
+  list = [];
+
+  constructor(list) {
+    this.list = list;
```

Comment:
You could also add a check for list to be of array type

Reply:
could, but don't think that it's needed to check types ;)